sdk/aqueduct/deserialize.py

```diff
@@ -34,11 +34,16 @@ def _read_bytes_content(content: bytes) -> bytes:
     return content
 
 
+def _read_none_content(content: bytes) -> bytes:
+    return content
```

Comment:
shouldn't this return None?

Reply:
To check my understanding, `content` is always an empty bytestring here right?

Reply:
yes, _read_none_content should always be a empty bytes(). And you are right on that _read_none_artifact should always return None